JPS_BASE_LIB/src/test/java/uk/ac/cam/cares/jps/base/annotate/test/MetaDataAnnotatorTest.java

```diff
@@ -0,0 +1,46 @@
+package uk.ac.cam.cares.jps.base.annotate.test;
+
+import java.util.Arrays;
+
+import org.junit.Assert;
+import org.junit.Test;
+
+import uk.ac.cam.cares.jps.base.annotate.MetaDataAnnotator;
+import uk.ac.cam.cares.jps.base.discovery.MediaType;
+
+public class MetaDataAnnotatorTest {
+
+	@Test 
+    public void testSparqlOverHttpService() {
+		Assert.assertNull(MetaDataAnnotator.getSparqlService());
+	}
+	
+	@Test
+	public void testGetSparqlInsertFull() {
```

Comment:
A HTTP request is made during this test (call to getInstance).  Could this be avoided by using a mock?